mirroroperator/registrymirror.py

```diff
@@ -375,7 +377,15 @@ def generate_stateful_set(self):
         mv $TEMPFILE {shared_cert_mount_path}/{cert_file}
         '''.format(upstream_cert_dir=UPSTREAM_CERT_DIR, cert_file=CERT_FILE,
                    shared_cert_mount_path=SHARED_CERT_MOUNT_PATH)
-
+        script_munge_nameservers = '''
+        cp /var/lib/conf.d/default.conf /var/lib/nameserversdirtemp/default.conf
+        NAMESERVERS=$(cat /etc/resolv.conf | grep "nameserver" | awk '{{print $2}}' | tr '\n' ' ')
+        if [ ! "$NAMESERVERS" == "" ]; then
+            sed -E -i "s/(resolver)/\1 $NAMESERVERS/" /var/lib/nameserversdirtemp/default.conf
+        else
+            sed -E -i "s/(resolver)/#\1/" /var/lib/nameserversdirtemp/default.conf
+        fi
+        '''
```

Comment:
I'm not sure why we are using a bash script in a container to edit a file that we are creating with a python template.

can we not just do this in python and put the value in the template like we do with `{{ auth }}`?

Reply:
this is also fixed...thanks